supervisely/nn/benchmark/base_benchmark.py

```diff
@@ -450,8 +466,16 @@ def visualize(self, dt_project_id=None):
         if dt_project_id is not None:
             self.dt_project_info = self.api.project.get_info_by_id(dt_project_id)
 
-        vis = Visualizer(self)
-        vis.visualize()
+        if self.visualizer_cls is None:
+            raise RuntimeError(
+                f"Visualizer class is not defined in {self.__class__.__name__}. "
+                "It should be defined in the subclass of BaseBenchmark (e.g. ObjectDetectionBenchmark)."
+            )
+        eval_result = self.evaluator.get_eval_result()

```

Comment:
- make `inference_info` optional and move `EvalResut` reading to visualizer
- mb rename `EvalResut`?